webextensions/api/extensionTypes.json

```diff
@@ -23,14 +19,58 @@
                 "version_added": "48"
               },
               "opera": {
-                "version_added": true,
-                "partial_implementation": true,
-                "notes": "This feature is supported but not exposed through the 'extensionTypes' object."
+                "version_added": true
               },
               "safari": {
-                "version_added": false,
-                "partial_implementation": true,
-                "notes": "This feature is supported but not exposed through the 'extensionTypes' object."
+                "version_added": false
```

Comment:
This would be `"14"`, because this `extensionTypes.ImageDetails` "virtual" type is used by `tabs.captureVisibleTab`, which in its turn is used by Safari.

ref:
- https://developer.apple.com/documentation/safariservices/safari_web_extensions/assessing_your_safari_web_extension_s_browser_compatibility#3584139
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/Tabs/captureVisibleTab#Browser_compatibility